GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -724,6 +724,13 @@ private void SetShortcutKeyDisplayStringsFromHotkeySettings()
         private void RefreshWorkingDirCombo()
         {
             var path = Module.WorkingDir;
+
+            // it appears at times Module.WorkingDir path is an empty string, this caused issues like #4874
+            if (string.IsNullOrWhiteSpace(path))
+            {
+                return;
```

Comment:
The `_NO_TRANSLATE_Workingdir.Text` should be updated to reflect that no repo is open.